pkg/kubefedctl/options/options.go

```diff
@@ -120,10 +117,14 @@ func GetScopeFromKubeFedConfig(hostConfig *rest.Config, namespace string) (apiex
 			Name:      util.KubeFedConfigName,
 		}
 		err = errors.Wrapf(err, "Error retrieving KubeFedConfig %q", config)
-		return "", err
+		return "", "", err
 	}
 
-	return fedConfig.Spec.Scope, nil
+	// use first 8 characters to identify resource for member cluster management
+	uid := string(fedConfig.UID)
```

Comment:
Given that the kubefed config can be removed and re-added, changing the uid, I suggest generating a uid and storing it as a field of the config. It can still be lost, but at least it won't be changed accidentally.